iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/metricsets/system/SystemMetrics.java

```diff
@@ -41,158 +35,114 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.Set;
-import java.util.concurrent.Future;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
 
 public class SystemMetrics implements IMetricSet {
   private static final Logger logger = LoggerFactory.getLogger(SystemMetrics.class);
   private static final String SYSTEM = "system";
   private final com.sun.management.OperatingSystemMXBean osMxBean;
-  private Future<?> currentServiceFuture;
-  private final ScheduledExecutorService service =
-      IoTDBThreadPoolFactory.newSingleThreadScheduledExecutor(
-          ThreadName.SYSTEM_SCHEDULE_METRICS.getName());
   private final Set<FileStore> fileStores = new HashSet<>();
-  private final boolean isDataNode;
-  private long systemDiskTotalSpace = 0L;
-  private long systemDiskFreeSpace = 0L;
-
-  public SystemMetrics(boolean isDataNode) {
-    this.isDataNode = isDataNode;
+  private final ArrayList<String> diskDirs;
+  private static final String NAME = "name";
+  private static final String SYS_CPU_LOAD = "sys_cpu_load";
```

Comment:
can we move `org.apache.iotdb.commons.service.metric.enums.Metric` and `org.apache.iotdb.commons.service.metric.enums.Tag`  to metric module?  What's your opinion @SpriCoder

Reply:
I think we can only move universal part into metric module